### PR TITLE
opt: add exploration steps to optstepsweb

### PR DESCRIPTION
**Pull request by RaduBerinde**
Adding visual `exploretrace` output to `optstepsweb`. The page now
shows a selector that can be toggled between normalization and
exploration.

Release note: None

### CODE REVIEW

File: pkg/sql/opt/testutils/opttester/opt_tester.go [hunk 1]
```diff
@@ -1331,8 +1331,25 @@ func (ot *OptTester) OptSteps() (string, error) {
 //
 // TODO(radu): currently, we only show normalization steps on this page.
```
Comment (DrewKimball) [inline pkg/sql/opt/testutils/opttester/opt_tester.go:1332]: [nit] you can remove this TODO now, right?

**Review by DrewKimball — APPROVED** [review]
This is so cool!

LGTM

**Review by RaduBerinde — COMMENTED** [review]
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/65874#-:-Mb-NAZR0LlhQYB0ml9R:b-ydi9ay)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @DrewKimball, @mgartner, and @rytaft)

___
*[pkg/sql/opt/testutils/opttester/opt_tester.go, line 1332 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/65874#-Maq0sSewkV4qtwJlPY0-r1-1332:-Mb-N8l79fLEw0hZbReK:b-qqx0w8) ([raw file](https://github.com/cockroachdb/cockroach/blob/ca72977bb9906ad0df4e5772e8e333c5cfe47762/pkg/sql/opt/testutils/opttester/opt_tester.go#L1332)):*
<details><summary><i>Previously, DrewKimball (Drew Kimball) wrote…</i></summary><blockquote>

[nit] you can remove this TODO now, right?
</blockquote></details>

Done. TFTR!

**Comment by rytaft** [comment]
Awesome!

**Comment by mgartner** [comment]
Very cool!

**Pull request merged**